CMakeLists.txt

```diff
@@ -269,7 +272,7 @@ add_custom_command(OUTPUT copy_test_reference_data
   )
 
 # put it there to avoid warnings from boost tests
-add_compile_options(-Werror) # warnings are errors!
+# add_compile_options(-Werror) # warnings are errors!
```

Comment:
Dunno, it got commented out somewhere in the murky mists of time. But yes, we should definitely keep this uncommented and _never commit anything that comments this out_ (looking at you, `$(git blame CMakeLists.txt:275)`

Reply:
```suggestion
add_compile_options(-Werror) # warnings are errors!
```

Reply:
I always comment this when I develop because this option makes coding even ore gruesome than it already is. It should always be uncommented on master though.